packages/core/src/modules/proofs/formats/indy/IndyProofFormatsServiceOptions.ts

```diff
@@ -24,12 +26,6 @@ export interface IndyVerifyProofFormat {
   proofRequest: Attachment
 }
 
-export interface IndyPresentationProofFormat {
```

Comment:
In the `IndyRequestedCredentialsFormat` these are all required, while they were optional in this interface. Is this going to lead to issues?

Reply:
I saw that and checked it, my thinking was that it was ok to use the same interface.